README.md

```diff
@@ -2,74 +2,81 @@
 
 ## Description
 
-This tool is a first draft of connecting an LMS interface to a Graylog server to create a dataset 
-of learning logs.
+This tool is dedicated to generate learning events (_aka_ tracking logs) from the Edx platform to feed dashboard development testing datasets.
```

Comment:
Maybe we should be more generalist and not restrict this project to openedx events. Tomorrow we may add a new LMS to the stack.